docs/framework/chat-assistant.mdx

```diff
@@ -2,153 +2,325 @@
+
+  When you click it, you'll see the **Properties** panel appear on the right side of the page. This lets you view and edit the properties of the selected component.
+
+  The first property you'll see in the panel is the **Text** property, which defines the text that appears as the header's title. It should contain the value `@{my_app.title}`. The `@{` and `}` indicate that `my_app.title` is a variable and that its contents should be the text displayed instead of the literal text "my_app.title". You'll set the value of this variable soon.
+  </Step>
+
+  <Step title="Clear the Section's default title">
+  Select the **Section** component by clicking it — it's just below the **Header** component and contains the title **Section Title** and a gray area labeled **Empty Section**.
 
-    The `Conversation` method can optionally accept a dictionary or a content prompt (e.g., “You are a social media expert in the financial services industry”), but it can also be left empty to use the defaults.
+  In the **Properties** panel, clear out the value of the **Title** property. This will remove the **Section**'s default title.
   </Step>
-  <Step title="Create a handler for the Chatbot component">
-    Next, create a handler for incoming messages by adding the `handle_simple_message` handler. This method will manage the chat interactions:
 
-    ```python
-    def handle_simple_message(state, payload):
-        state["conversation"] += payload
-        
-        for chunk in state["conversation"].stream_complete():
+  <Step title="Add a Text component to the Section">
+  Drag a **Text** component from the **Toolkit** panel on the left (it's under **Content**, and you may need to scroll down a little to find it) and into the *Section*. Sections can act as containers for other components.
+
+  Select the **Text** component. In the **Properties** panel, set the **Text** property to provide instructions or context for your chat assistant. Here's an example: `Welcome to the Chat Assistant. Ask me anything!`
+  </Step>
+
+  <Step title="Add a Chatbot component to the Section">
+  The heart of this app is the **Chatbot** component, a pre-built component that displays the conversation between the LLM and the user and provides a text field where the user can enter prompts.
+
+  Drag a **Chatbot** component from the **Toolkit** panel (it's under **Content**) into the *Section*, just below the Text box.
+  </Step>
+</Steps>
+
+The project editor should now look like this:
+
+![Project editor with components laid out](/framework/images/tutorial/chat/chat_assistant_2c.png)
+
+
+## Add the back-end code
+
+With the UI laid out, it's time to work on the logic behind it. 
+
+Switch to your code editor and open the `main.py` file in your project's directory. This file was automatically generated; you'll update the code in it to define the behavior of your app.
```

Comment:
Add something about "or switch to the in-browser editor by clicking the button on the top right"

Reply:
Updated.